### PR TITLE
Added OpenGraph meta tag to login.html

### PR DESCRIPTION
**Pull request by PLOKMJNB**
discard if not useful

### CODE REVIEW

**Comment by lze3** [comment]
what's the purpose of this?

**Comment by PLOKMJNB** [comment]
picture in the meta tag, as I said useless.

File: web/basic/login.html [hunk 1]
```diff
@@ -6,9 +6,25 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
-    <meta name="description" content="txAdmin - remotely Manage & Monitor your GTA5 FiveM Server">
+    <title>txAdmin | {{it.headerTitle}}</title>
```
Comment (EvanDarwin) [inline web/basic/login.html:9]: This should be `{{it.headerTitle}} | txAdmin`

Comment (PLOKMJNB) [reply web/basic/login.html:9]: guess so

**Comment by manzarek123** [comment]
SEO is important for txAdmin /s
The changes are probably good tho :)

**Comment by tabarra** [comment]
OG is cool, add more things to it!
```
{{it.serverProfile}}
{{it.txAdminVersion}}
{{it.fxServerVersion}}
```

**Comment by PLOKMJNB** [comment]
ok Tabby Wabby

File: web/basic/login.html (hunk 1 above)
Comment (tabarra) [reply web/basic/login.html:9]: Actually no, header title logic is in the rendered function

**Pull request merged**